code/dce-downloader/node/README.MD

```diff
@@ -0,0 +1,233 @@
+# DCE Downloader Tools in Node.js
+
+Two Node.js scripts files are provided to get data from the Bazaarvoice Displayable Content Export service. Please refer to [documentation](https://developer.bazaarvoice.com/displayable-content-export) for details.
+
+## Prerequisite
+1. You should have Node.js 5.0 or later installed.
```

Comment:
what was the reasoning for supporting as low as Node 5? AWS lambda supports up to Node v6.10. I think we should try to at least keep up with AWS in terms of versioning.